src/index.js

```diff
@@ -67,6 +67,7 @@ const PUSH_PREF = 'bespiral.push.pref'
 const SELECTED_COMMUNITY_KEY = 'bespiral.selected_community'
 const env = process.env.NODE_ENV || 'development'
 const config = configuration[env]
+console.log(flags())
```

Comment:
I guess we don't need this debugging info anymore.

Reply:
Good catch!